content/applications/hr/time_off.rst

```diff
@@ -0,0 +1,603 @@
+  end date. The selected start and end dates appear in deep purple, and the dates between them
+  appear in pale purple (if applicable). If the time off requested is for a single day, click on the
+  start date, then click the same date again for the end date. When the correct dates are
+  selected/highlighted, click the :guilabel:`Apply` button. The dates now populate the
+  :guilabel:`From` and :guilabel:`To` fields.
+
+  - :guilabel:`Half Day`: if the time off request is for a half day, activate this box. When this is
+    selected, the :guilabel:`From` date field disappears, and is replaced with a drop-down menu.
+    Select either :guilabel:`Morning` or :guilabel:`Afternoon` to indicate which half of the day is
+    requested.
+  - :guilabel:`Custom Hours`: if the time off requested is not a whole or half day, activate this
+    box. A :guilabel:`From` and :guilabel:`To` field appears beneath this option if selected. Using
+    the drop-down menu, select the start and end time for the time off request.
+
+- :guilabel:`Duration`: this field updates automatically once the :guilabel:`Date` section is
+  completed. If the :guilabel:`Date` section is modified, this section automatically updates to
+  reflect toe total time off requested. This field is in either hours or days, depending on the
+  :guilabel:`Date` selections.
+- :guilabel:`Description`: enter a description for the time off request. This should include any
+  details that the manager and approvers need in order to approve the request.
+- :guilabel:`Supporting Document`: if the :guilabel:`Time Off Type` selected allows for the
+  attachments of documents, this field appears. Click the :guilabel:`Attach File` button and a file
+  explorer window appears. Navigate to the file(s) to attach, then click the :guilabel:`Open`
+  button. The files then appear on the time off request form. Multiple documents can be attached.
+
+When the form is complete, click the :guilabel:`Save` button to save the information and submit the
+request.
+
+.. image:: time_off/time-off-request.png
+   :align: center
+   :alt: A time off request form filled out for an employee home sick for two days with the flu.
+
+.. _time_off/request-allocation:
+
+Request allocation
+==================
+
+If an employee has used all their time off, or is going to run out of time off, they can request
+an allocation for additional time. Allocations can be requested in one of two ways, either from the
+:ref:`dashboard <time_off/dashboard>` or the :guilabel:`My Allocations` view.
+
+To access the dashboard, navigate to the :menuselection:`Time Off application --> My Time Off -->
+Dashboard`. This is also the default view for the *Time Off* application.
+
+To access :guilabel:`My Allocations`, navigate to the :menuselection:`Time Off application --> My
+Time Off --> My Allocations`. This presents a list view of all the allocations for the employee.
```

Comment:
I combined the paragraph but not the directions, because I wanted to have the 'default view' line not be looked over and make sense. I also like to keep all the directions spelled out- even if it seems like overkill, so no one gets confused.